analytics_data_api/v0/views/learners.py

```diff
@@ -28,7 +28,7 @@
 
 class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
     """
-    Get a particular student's data for a particular course.
+    Get data for a particular learner in a particular course.
```

Comment:
That makes sense to me. I noticed that the term "user" is being used liberally throughout - for example on line 41 below where it says `"username: User's username."`. Should that read `"username": "Learner's username"`? I don't mind either way.